pkg/cmd/util/output/backup_describer.go

```diff
@@ -87,6 +91,13 @@ func DescribeBackup(
 			d.Println()
 			DescribePodVolumeBackups(d, podVolumeBackups, details)
 		}
+		// TODO(nrb-csi): Put the CSI description here? Or down with the DescribeBackupStatus?
```

Comment:
I *think* this TODO and the below "scenarios" comment can be removed now, but correct me if I'm wrong.